src/components/VaultRow/lib/VaultTable.tsx

```diff
@@ -6,28 +6,28 @@ import { TxSettingsPopover } from "@/components/TxSettingsPopover"
 
 type VaultTableProps = {
   label: string
-  earningEnabled: boolean
+  showEarningsColumn: boolean
 }
 
 export const VaultTable: FC<PropsWithChildren<VaultTableProps>> = ({
   children,
   label,
-  earningEnabled,
+  showEarningsColumn,
 }) => {
   return (
     <Table>
       <div className="relative z-[1] max-lg:hidden" role="rowgroup">
         <TableRow
           className={clsx(
             "overflow-visible rounded-b-none border-b border-b-pink/30",
-            earningEnabled ? "" : "lg:grid-cols-[4fr,1fr,1fr,1fr,3.5rem]"
+            showEarningsColumn ? "" : "lg:grid-cols-[4fr,1fr,1fr,1fr,3.5rem]"
```

Comment:
Few things here:

1. You should update this to use `clsxm` instead of raw `clsx`
2. Because this component is responsible for *adding an additional column*, I think it might make more sense to structure the classnames in the same way. The base TableRow should support the standard 5 columns with the classname here overriding it to support 6 columns based on this prop.
3. This is the wrong way to use `clsxm`, should be done as such:

```ts
clsx("these class-names always are applied", {
  "conditional-classname": isSomeCondition,
  "another-conditional-classname": !isSomeCondition
})
```